src/main/java/http/ClientHandler.java

```diff
@@ -5,29 +5,30 @@
 
 public class ClientHandler {
     private final Socket client;
-    private final RequestHandler requestHandler;
-
-    private final ResponseHandler responseHandler;
 
     public ClientHandler(Socket socket) {
         client = socket;
-        requestHandler = new RequestHandler();
-        responseHandler = new ResponseHandler();
     }
 
     public void start() {
         try {
             InputStream in = client.getInputStream();
             ByteArrayOutputStream out = new ByteArrayOutputStream();
-            String request = requestHandler.getRequest(in);
-            System.out.println(request);
 
-            out.write(responseHandler.getPageNotFoundResponse());
+            String request = RequestHandler.getRequest(in);
+            String parameters = RequestHandler.getRequestParameters(request);
```

Comment:
refactor: This would be a good place to add a `Request` class which contains getter/setters 

For example: 

```
public class Request {
    private final HttpMethod httpMethod;
    private final String parameters;
    private final String method;
    private final String path;
    private final String requestBody;

    public Request(method, path, requestBody) {
        // 
    }

    public HttpMethod getHttpMethod() {
        return httpMethod;
    }

```

Then have a `parseRequest` method which returns a `request`
```
public Request parseRequest(String parameters) throws IOException {
     // get all parts

        return buildRequest(method, path, body);
    }
```